.github/workflows/optimization_integration.yml

```diff
@@ -0,0 +1,175 @@
+name: Optimization Integration tests
```

Comment:
Can we migrate the rest of the FML tests from LMI-Distro and make this the one stop shop for all FML tests?